### DISCUSSION TITLE
Zigbee Device that dont like all other devices

### DISCUSSION THREAD

**Discussion by MattWestb** [zigpy/zigpy#575] [General]
Some Zigbee devices with or without Zigbee certificat can having problems living together with certain devices of some reasons.

One of the most famous is the OSRAM smart + Outdoor plug (that i like very much but have not buying) that have bad antenna and buggy firmware that also can corrupting package so its children and / or neighbours getting problem forwarding messages to / from them. Normally Xiaomi sensors dont like that and is leaving the network without trying rejoining other routers.

Older Philips HUE lights (I only have one) is ZLL (not ZB3) and if some end device is changing to one other parent its not timing out the child and reporting its having it in the neighbor table for weeks or longer. To getting around that i resetting the bulb and rejoining it and its working normal (until next end device is moving from it).
Then its only ZLL (probably not Zigbee PRO) its only using the coordinator as parent and not trying routing traffic around the mech.

Philips HUE end devices / controlling devices.
I have one HUE Dimmer Switch that is working great after fixing the binding problem.
And 2 HUE motion sensors. They have starting living and joining the network and moving to very crazy routers (one in the living room have moved to one IKEA outlet in the kitchen that is 10m away with wors signal you can find in the network). And also one have 80% battery after over 2 years and the other only 18% battery. I think the first is reported wrong (80%) but i have not investigating it yet.

LIDL / tuya ZB3 devices. Is Zigbee 3 (Silabs chip) and the router devices looks liking other LIDL routers as parent instead of the coordinator (IKEA Billy EZSP = Silabs) comparing with IKEA routers that is only having the coordinator as parent. 
It can being very good if its because its the best route to the coordinator or it if it becos ZCL ver = 3 in basic cluster then its bad. I can being that tuya is using one newer stack as IKEA but that is not confirmed.
The routers is trying building connections to all possible routers and having they open for traffic ala ZB3.

IKEA routers. Nearly all devices have being updated to ZB3 but not the old motion sensor and some ZLL lights like My old E27 CWS.
Both ZLL and ZB3 routers is trying building connections to all possible routes and having they open for traffic ala ZB3. Some Xiaomi sensors didn't like being forces to paring via ZB3 routers but was ok with ZLL ones, but after being paired i was tricking them to move to ZB3 routers and its working well and have not leaving the network for 3 months.

I hoping more users is sharing their experience of devices that is liking or disliking other devices so user can shusing devices that not having problems with each other in their systems.

If its going well its very good putting then on one wiki page as reference for system deployment.

Also taking one look at the WiKi [ZHA-Supported-Devices](https://github.com/zigpy/zigpy/wiki/ZHA-Supported-Devices)

Thanks in advance for sharing your experience with the community !!

**Comment by Hedda** [comment]
Xiaomi and Aqara also have a lot of issues and the Hubitat have collected a lot of tips and best practices about how to use them:

https://community.hubitat.com/t/xiaomi-aqara-devices-pairing-keeping-them-connected/623

**Reply by mguaylam** [reply]
I hope their T1 series will put all of this behind us. Just unfortunate its very hard to find or near impossible.
